src/MarketingPageAcceptanceTests.TestData/Utils/Queries.cs

```diff
@@ -7,6 +7,7 @@ internal static class Queries
         internal const string UpdateSolutionSolutionDetailId = "UPDATE Solution SET SolutionDetailId=@solutionDetailId WHERE Id=@solutionId";
         internal const string UpdateSolutionSupplierlId = "UPDATE Solution SET SupplierId=@supplierId WHERE Id=@solutionId";
         internal const string DeleteSolution = "DELETE from Solution where Id=@solutionId";
+        internal const string GetAllSolutionIds = "Select Id from [dbo].[Solution]";
```

Comment:
can we change this to only return Ids which contain the prefix?

Reply:
not a bad call, I'll get on it